subworkflows/ebi-metagenomics/reads_bwamem2_decontamination/tests/nextflow.config

```diff
@@ -0,0 +1,6 @@
+process {
+    withName: BWAMEM2_MEM {
+        ext.args = "-M"
+        ext.args2 = "-f 4 -F 256 -uS"
+    }
+}
```

Comment:
if this config is not any different that the one in the main folder of the module, I would remove it and call the other one instead in the main.nf.test:
config "subworkflows/ebi-metagenomics/reads_bwamem2_decontamination/nextflow.config"
instead of:
config "subworkflows/ebi-metagenomics/reads_bwamem2_decontamination/tests/nextflow.config"